### DISCUSSION TITLE
QFile - provide a possibility to customize the behavior when drag n drop

### DISCUSSION THREAD

**Discussion by robert-niestroj** [quasarframework/quasar#13629] [Ideas / Proposals] [answered]
Hi,
as for now when a file is being dragged on a QFile it shows insetted dashed border. Please provide a possibility to customize this. I'd like to change the border and the background-color. CSS Classes to do this would be ok.

**Comment by pdanpdan** [comment, marked as answer]
https://codepen.io/pdanpdan/pen/oNEPWPO

**Reply by robert-niestroj** [reply]
![image](https://user-images.githubusercontent.com/9776384/172375201-714a7883-e509-4f1b-961f-7636b5c521a0.png)

It's a bit hard to style. This q_file_dnd class is a sibling to the rest. When i change his bg-color to gray the "drag and drop or click to upload" disappear beacuse his is gray too. There is no previous sibling css selector so it's hard to catch this text. Could the dnd class be applied to the top label tag?

**Reply by pdanpdan** [reply]
I think a class on QFile / QUploader would be better - I'll think about it

**Reply by robert-niestroj** [reply]
> 

Yes, i think QFile is compiled to this html label tag. I tried to circumvent the Problem with z-index: -10000 but that stopped drap&drop from working. So this class on QFile seems to be the best solution.

**Comment by rstoenescu** [comment]
Will be available in Quasar v2.7.2 and v1.19.2
